toptal/picasso
# feat: add text selection on focus to Select component

[SPC-443](https://toptal-core.atlassian.net/browse/SPC-443)

### Description

Clicking a Select field should select the current text in its Input by default

### How to test

- Go to Select component in Storybook
- Select an option in Default case
- Focus on the Default case Select field again, text should be selected

### Review
